packages/babel-plugin-polyfill-record-tuple/test/fixtures/usage-pure/json-parse-immutable/output.mjs

```diff
@@ -0,0 +1,3 @@
+import { parseImmutable as _parseImmutable } from "@bloomberg/polyfill-record-tuple";
```

Comment:
This is not really pure, it stills define `JSON.parseImmutable = ...`. Imo the polyfill should support another entry point that doesn't define the globals but just exports them.